modules/attributes/pages/document-attributes-reference.adoc

```diff
@@ -36,6 +37,229 @@ Otherwise, the assignment won't have any effect on the document.
+
+|docdatetime
+|_datetime (ISO)_ +
+_ex._ 2019-01-04 19:26:06 UTC
+|{y}
+|{n}
+|Last modified date and time of the source document.^[<<note-docdatetime,1>>,<<note-sourcedateepoch,2>>]^
+
+|docdir
+|_directory path_ +
+_ex._ /home/user/docs
+|If input is a string
+|{y}
+|Full path of the directory that contains the source document.
+Empty if the safe mode is SERVER or SECURE (to conceal the file's location).
+
+|docfile
+|_file path_ +
+_ex._ /home/user/docs/userguide.adoc
+|If input is a string
+|{y}
+|Full path of the source document.
+Truncated to the basename if the safe mode is SERVER or SECURE (to conceal the file's location).
+
+|docfilesuffix
+|_file extension_ +
+_ex._ .adoc
+|If input is a string
+|{y}
+|File extension of the source document, including the leading period.
+
+|docname
+|_file stem basename_ +
+_ex._ userguide
+|If input is a string
+|{y}
+|Root name of the source document (no leading path or file extension).
+
+|doctime
+|_time (ISO)_ +
+_ex._ 19:26:06 UTC
+|{y}
+|{n}
+|Last modified time of the source document.^[<<note-docdatetime,1>>,<<note-sourcedateepoch,2>>]^
+
+|docyear
+|_integer_ +
+_ex._ {docyear}
+|{y}
+|{n}
+|Year that the document was last modified.^[<<note-docdatetime,1>>,<<note-sourcedateepoch,2>>]^
+
+|embedded
+|_empty_
+|{n}
+|n/a
+|Only set if content is being converted to an embedded document (i.e., body of document only).
+
+|filetype
+|_any_ +
+_ex._ html
+|If input is a string
+|{y}
+|File extension of the output file name (without leading period).
+
+|htmlsyntax
+|`html` +
+`xml`
+|{n}
+|n/a
+|Syntax used when generating the HTML output.
+Controlled by and derived from the backend name (html=html or xhtml=html).
+
+|localdate
+|_date (ISO)_ +
+_ex._ 2019-02-17
+|{y}
+|{n}
+|Date when the document was converted.^[<<note-sourcedateepoch,2>>]^
+
+|localdatetime
+|_datetime (ISO)_ +
+_ex._ 2019-02-17 19:31:05 UTC
+|{y}
+|{n}
+|Date and time when the document was converted.^[<<note-sourcedateepoch,2>>]^
+
+|localtime
+|_time (ISO)_ +
+_ex._ 19:31:05 UTC
+|{y}
+|{n}
+|Time when the document was converted.^[<<note-sourcedateepoch,2>>]^
+
+|localyear
+|_integer_ +
+_ex._ {localyear}
+|{y}
+|{n}
+|Year when the document was converted.^[<<note-sourcedateepoch,2>>]^
+
+|outdir
+|_directory path_ +
+_ex._ /home/user/docs/dist
+|{n}
+|n/a
+|Full path of the output directory.
+(Cannot be referenced in the content.
+Only available to the API once the document is converted).
+
+|outfile
+|_file path_ +
+_ex._ /home/user/docs/dist/userguide.html
+|{n}
+|n/a
+|Full path of the output file.
+(Cannot be referenced in the content.
+Only available to the API once the document is converted).
+
+|outfilesuffix
+|_file extension_ +
+_ex._ .html
+|{y}
+|{n}
+|File extension of the output file (starting with a period) as determined by the backend (`.html` for `html`, `.xml` for `docbook`, etc.).
+
+|safe-mode-level
+|`0` +
+`10` +
+`20` +
+`30`
```

Comment:
It should be `0`, `1`, `10` and `20` no?

Reply:
Good catch!